### PR TITLE
namespace loggers #18

### PR DESCRIPTION
**Pull request by crookse**
https://github.com/drashland/deno-drash/issues/18

### CODE REVIEW

File: mod.ts [hunk 1]
```diff
@@ -66,13 +66,20 @@ namespace Drash {
...
     export type FileLogger = file_logger;
     export const FileLogger = file_logger;
     export type Logger = base_logger;
     export const Logger = base_logger;
+  };
+
+  export const Loggers: any = {};
+
+  export function addLogger(name: string, logger: any) {
+    // todo: throw error
```
Comment (crookse) [inline mod.ts]: with todo statments, i go with the what deno is doing as a standard. example:

```
TODO(crookse) throw error
```

Comment (crookse) [reply mod.ts]: im not saying you have to conform to it, but it's nice to know who wrote it in case it's a long-running TODO

Comment (saragee3) [reply mod.ts]: Good point. I'll keep that in mind.

**Comment by crookse** [comment]
lgtm

**Comment by crookse** [comment]
documentation needs to be updated in `deno-drash-docs` repo now. i'll guide you through that repo

**Pull request merged**